lighthouse-core/report/html/templates.html

```diff
@@ -664,10 +690,13 @@
 
   </style>
   <a href="#" class="lh-gauge__wrapper">
-    <svg viewBox="0 0 120 120" class="lh-gauge">
-      <circle class="lh-gauge-base" r="56" cx="60" cy="60"></circle>
-      <circle class="lh-gauge-arc" transform="rotate(-90 60 60)" r="56" cx="60" cy="60"></circle>
-    </svg>
+    <!-- Wrapper exists for the ::before plugin icon. Cannot create pseudo-elements on svgs. -->
```

Comment:
HTML comments in templates end up in the final report DOM so...

it'd be slightly better to have them on the outide of the <template> (where they are hardly any use.)  sucks as I love having these, but they're awkward additions to our shipped DOM.

Reply:
fuck it lets keep these and remove comment nodes from DOM.cloneTemplate() in a separate PR

Reply:
fudge it #8390